src/zodbpickle/tests/pickletester_3.py

```diff
@@ -1112,7 +1111,7 @@ def test_bad_getattr(self):
             self.assertRaises(RuntimeError, self.dumps, x, proto)
 
         # protocol 2 don't raise a RuntimeError, except under PyPy, or Python >= 3.4
-        if _is_pypy or (_is_pure and not _PY33) or _PY34:
+        if _is_pypy or _PY34:
```

Comment:
I'm not sure it was right to just remove the condition `(_is_pure and not _PY33)`. The condition `not _PY33` always simplifies to `True` (because this code is only run on Python 3, see `test_pickle.py`, and now only on Python 3.4+). So the original condition became `_is_pypy or (_is_pure and True) or _PY34`, which becomes `_is_pypy or _is_pure or _PY34`. `_PY34` is always true now, so this should just be `_is_pypy or _is_pure`. 

This suggests to me that we're not testing the pure-python code on CPython (which is something to fix in another PR). Or, if we are, then this entire condition is irrelevant and misleading and can go away.

Reply:
Argh, wait, I think I missed a not. `not _PY33` always becomes `False`. So `_is_pure and False` becomes `False`, and so `_is_pypy or False or _PY34` becomes just `_is_pypy`.

Reply:
But `_PY34` is now always `True` so the whole condition becomes `True` thus irrelevant. (If we require Python 3.4+, the used PyPy3 version should also a version which supports 3.4+.)

Reply:
Ah, true, since this file is only imported and used by Python 3, `_is_pypy` is redundant.